packages/api/src/hooks/UseCennznet.ts

```diff
@@ -13,10 +13,7 @@
 // limitations under the License.
 
 import { web3Accounts, web3Enable } from '@polkadot/extension-dapp';
```

Comment:
should publish the @cennznet/ versions sometime soon

Reply:
ya to npm registry..